docs/resources/job.md

```diff
@@ -0,0 +1,91 @@
+# databricks_job Resource
+
+The databricks_job resource allows you to create, edit, and delete jobs, which run on either new or existing [clusters](cluster.md).
+
+## Example Usage
+
+```hcl
+resource "databricks_job" "this" {
+    name = "Featurization"
+    timeout_seconds = 3600
+    max_retries = 1
+    max_concurrent_runs = 1
+    
+    new_cluster  {
+        num_workers   = 300
+        spark_version = "6.6.x-scala2.11
+        node_type_id  = "i3.xlarge"
+    }
+    
+    notebook_task {
+        notebook_path = "/Production/MakeFeatures"
+    }
+    
+    libraries {
```

Comment:
@stikkireddy i think we'd need one more PR to change `libraries` to `library` in jobs & clusters. There's also a bit of resource inconsistency - `library` blocks logically should be within `new_cluster` block. though, JAR jobs on existing clusters may think otherwise.

Reply:
@nfx first statement makes sense its awkward to say libraries but only define one. We should make the change before pushing in this release then. The second statement is inaccurate as you can define jobs onto an existing cluster but the jobs can still install libraries. So it does not make sense for library construct to be built into the new_cluster.